### PR TITLE
RDM-5544- ACR references changes to hmctspublic and hmctsprivate

### PR DESCRIPTION
**Pull request by akdasari**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: compose/backend.yml [hunk 1]
```diff
@@ -91,9 +91,9 @@ services:
       - service-auth-provider-api
 
   sidam-api:
-    image: hmcts.azurecr.io/hmcts/sidam/idam-api:1.0.0
+    image: "hmctspublic.azurecr.io/idam/api:pr-350"
     environment:
-      JAVA_OPTS: -Xms128m -Xmx512m -Dlogging.level.uk.gov.hmcts=DEBUG
+      JAVA_OPTS: -Xms128m -Xms512m -Dlogging.level.uk.gov.hmcts=DEBUG
```
Comment (smathangi) [inline compose/backend.yml]: revert this otherwise @mario-paniccia  will after you 🏃

File: compose/frontend.yml [hunk 2]
```diff
@@ -55,17 +55,17 @@ services:
...
   idam-web-public:
-    image: "hmcts.azurecr.io/hmcts/sidam/idam-web-public:1.0.0"
+    image: "hmctspublic.azurecr.io/idam/web-public:latest"
     environment:
      STRATEGIC_SERVICE_URL: http://sidam-api:5000
      IDAM_CONTINUE_URL_VALIDATOR_ENFORCE_TLS: "false"
      REFORM_SERVICE_NAME: sidam-api
      REFORM_TEAM: idam
      REFORM_ENVIRONMENT: local
-     JAVA_OPTS: -Xms128m -Xmx512m
```
Comment (smathangi) [inline compose/frontend.yml]: keep this

File: compose/backend.yml [hunk 3]
```diff
@@ -126,23 +126,20 @@ services:
       TESTING_SUPPORT_ENABLED: "true"
 
   fr-am:
-    image: hmcts.azurecr.io/hmcts/sidam/sidam-fr-am:1.0.0
-    environment:
-      JAVA_OPTS: -Xms128m -Xmx512m
```
Comment (smathangi) [inline compose/backend.yml]: same as above keep these

File: compose/frontend.yml [hunk 4]
```diff
@@ -43,9 +43,9 @@ services:
       - ccd-data-store-api
 
   idam-web-admin:
-    image: hmcts.azurecr.io/hmcts/sidam/idam-web-admin:1.0.0
+    image: "hmctspublic.azurecr.io/idam/web-admin:latest"
     environment:
-      JAVA_OPTS: -Xms128m -Xmx512m
+      JAVA_OPTS: -Xms128m -Xms512m
```
Comment (smathangi) [inline compose/frontend.yml]: same as above

File: compose/backend.yml [hunk 5]
```diff
@@ -91,9 +91,9 @@ services:
       - service-auth-provider-api
 
   sidam-api:
-    image: hmcts.azurecr.io/hmcts/sidam/idam-api:1.0.0
+    image: "hmctspublic.azurecr.io/idam/api:pr-350"
```
Comment (mario-paniccia) [inline compose/backend.yml]: how come we are using a branch of idam?

File: compose/backend.yml (hunk 1 above)
Comment (mario-paniccia) [reply compose/backend.yml]: Anand what's the rationale of this change please?

File: compose/backend.yml (hunk 3 above)
Comment (mario-paniccia) [reply compose/backend.yml]: Anand what's the removal of JAVA_OPTS please?

File: compose/backend.yml [hunk 6]
```diff
@@ -126,23 +126,20 @@ services:
...
 
   fr-idm:
-    image: hmcts.azurecr.io/hmcts/sidam/sidam-fr-idm:1.0.0
+    image: "hmctsprivate.azurecr.io/idam/idam-fr-idm:latest"
     ports:
       - 18080:18080
       - 9010:9010
     environment:
       JAVA_TOOL_OPTIONS: -Dcom.sun.management.jmxremote.port=9010 -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=vm-ubuntu16 -Dcom.sun.management.jmxremote.rmi.port=9010
-      JAVA_OPTS: -Xms128m -Xmx512m
```
Comment (mario-paniccia) [inline compose/backend.yml]: Anand what's the removal of JAVA_OPTS please?

File: compose/backend.yml [hunk 7]
```diff
@@ -165,23 +162,25 @@ services:
...
-      AM_DB:
-      AM_DB_USERNAME:
-      AM_DB_PASSWORD:
-    healthcheck:
-      interval: 10s
-      timeout: 10s
-      retries: 10
-    ports:
-      - 5600:5432
+  # am-database:
```
Comment (mario-paniccia) [inline compose/backend.yml]: please add a comment explaining that this commenting out is just temporary and the reason

File: compose/backend.yml [hunk 8]
```diff
@@ -165,23 +162,25 @@ services:
       - ccd-docker-ccd-shared-database-data:/var/lib/postgresql/data
 
   shared-db:
-    image: hmcts.azurecr.io/hmcts/sidam/sidam-idm-pg:1.0.0
+    image: "hmctsprivate.azurecr.io/idam/shared-db:latest"
     ports:
       - 5051:5432
+    volumes:
+      - ccd-docker-idam-shared-database-data:/var/lib/postgresql/data
```
Comment (mario-paniccia) [inline compose/backend.yml]: What's the rationale of this change? please avoid changes not related to the ticket scope.
We already have a branch where we introduced a volume to sidam db. That branch is currently under testing.

File: compose/backend.yml [hunk 9]
```diff
@@ -195,3 +194,4 @@ services:
 volumes:
   ccd-docker-ccd-shared-database-data:
   wiremock:
+  ccd-docker-idam-shared-database-data:
```
Comment (mario-paniccia) [inline compose/backend.yml]: relevant for this ticket?

File: compose/frontend.yml (hunk 4 above)
Comment (mario-paniccia) [reply compose/frontend.yml]: yes please revert this change or explain why has been introduced

File: compose/backend.yml (hunk 1 above)
Comment (akdasari) [reply compose/backend.yml]: @mario-paniccia  That change wasn't in master, when i updated yesterday morning
Did your changes made to master?

File: compose/backend.yml (hunk 3 above)
Comment (akdasari) [reply compose/backend.yml]: @mario-paniccia  Not sure who made those changes, as I mentioned above it wasn't from me.

File: compose/backend.yml (hunk 5 above)
Comment (akdasari) [reply compose/backend.yml]: @mario-paniccia  , that's the new location for sidam image, and also as per SIDAM team we have to use PR for now, and will change to latest when they push proper tagging

File: compose/backend.yml (hunk 8 above)
Comment (akdasari) [reply compose/backend.yml]: @mario-paniccia  just thought of improving this, sorry didn't see other PR, will revert
